### ISSUE TITLE
PUBGSharpException > 'Player data is not valid....'

### ISSUE THREAD

**Issue by Arrowka** [eklypss/PUBGSharp#27]
Hi!

First of all, i wanna say thank you to this nugget, but unfortunatelly not working for me. :( I have api key, but GetPlayerStatsAsync result's is always this : 'Player data is not valid.. Player might not exist, or their stats have not been updated yet.' What's the problem?

**Comment by CGKajm** [comment]
It's not the PUBGSharp fault, although it could print the error message. The problem is that the PUBGTracker API is down. And it's been down for over a month.  There is still a way go to get the data, but that's something you have to figure out.

**Comment by Arrowka** [comment]
Hm.. Are you sure of it? The WebBrowser give me the following result: {"message":"No API key found in request"}. This points to the fact that the API is working.

**Comment by CGKajm** [comment]
I am sure, maybe try it yourself? You'll get json file containing "error" : "Api is disabled because of a throttling we get from Bluehole... .... ".

**Comment by PoweredbyDoS** [comment]
Yes, as **CGKajm** advised this service is 100% Reliant on the Pubgstats API, when it's down you wont get data. You also can't simply put the request link in the address bar of your web browser, you need to pass the API-Key in headers. 

> ![image](https://user-images.githubusercontent.com/36194055/35859451-3a416a4a-0b94-11e8-86cb-54e1ec2316e0.png)`

Unfortunately PUBGTracker is less than forthcoming in exactly what is going on or eta of outage, however as mentioned apparently it is out of their hands due to throttling direct with Bluehole and no doubt their own website takes their priority.

I pass errors onto response to users.
![image](https://user-images.githubusercontent.com/36194055/35859889-fbfc1a4e-0b95-11e8-84d7-766ca83f96fc.png)

Hoping they get this sorted sooner rather than later!

**Comment by EolmY** [comment]
It's sad that bluehole is throttling on they end.

I recently found out that pubg.op.gg also has a api. Its far more smaller but it feature the overall stats but no history if i see it right.

I created a netcore library for it on my repo. If you find it interesting and fight new endpoints feel free to contribute

**Comment by CGKajm** [comment]
There is also still a way to get data from PUBGTracker, but if pubg.op.gg has API now, that's cool

**Comment by EolmY** [comment]
Looked again in and Found the Game History Endpoint and the Deathlist for each game in cronologic order. Pushed it already

https://github.com/EolmY/PUBGOPGG

**Comment by Arrowka** [comment]
Thank you! I will chek it out soon.

**Comment by Arrowka** [comment]
Thank You! Its Working Fine!

**Comment by Arrowka** [comment]
Me again! :D One question. I cant find Team mates name in the datas. Can you help me how can i find it?
